api/v1/verticadb_webhook.go

```diff
@@ -1828,20 +1828,18 @@ func (v *VerticaDB) checkAnnotatedSubclustersInShutdownSandbox(oldObj *VerticaDB
 	for sandboxName := range persistSandboxes {
 		newSandbox := newSandboxMap[sandboxName]
 		oldSandbox := oldSandboxMap[sandboxName]
-		if newSandbox.Shutdown {
+		if newSandbox.Shutdown { // !oldSandbox.Shutdown &&
```

Comment:
Why this comment? I do not think oldSandbox.shutdown value matters.

Reply:
This is useless. I will remove it.